OpenOversight/app/main/forms.py

```diff
@@ -197,6 +205,12 @@ class DepartmentForm(Form):
         default="",
         validators=[Regexp(r"\w*"), Length(max=100), DataRequired()],
     )
+    state = SelectField(
+        "The law enforcement agency's home state",
+        coerce=str,
+        choices=get_state_choices(),
+        validators=[Length(min=0, max=2), DataRequired()],
+    )
```

Comment:
I'll scrape through these docs tomorrow and get this solved: https://wtforms-sqlalchemy.readthedocs.io/en/latest/wtforms_sqlalchemy/